### DISCUSSION TITLE
Adding a negative horizontal indent for a tabularray table

### DISCUSSION THREAD

**Discussion by russurquhart1** [lvjr/tabularray#508] [Q&A]
I'm writing errata topics that have a very simple format:

```
\documentclass[oneside]{book}
\usepackage{tabularray}
\begin{document}

\subsection{Software blows up after starting} \label{JIRA-3079}

\begin{tblr}{>{\bfseries}ll}
Errata ID & JIRA-3079 \\
Functional Area & System OS \\
Component & NOC \\
Severity & S5: No workaround \\
\end{tblr}
\end{document}
```
I wanted to keep the coding of the table simple. My only issue is that the table is indented about 2.5 mm in from the left margin, and doesn't line up with the subsection title above it. Short of putting a `\hspace{-2.5mm}` in front of each row, is there a more elegant wave to more the table to the left by -25mm?

**Comment by muzimuzhi** [comment]
It's caused by the `leftsep` of first column, which defaults to `6pt` (~= 2.11mm).

Setting inner option `column{1}={leftsep=0pt}` does the trick, meanwhile the table preamble has to be specified by `colspec={>{\bfseries}ll}`.

```tex
\documentclass[oneside]{book}
\usepackage{tabularray}
\begin{document}

% before
\subsection{Software blows up after starting} \label{JIRA-3079}

\begin{tblr}{>{\bfseries}ll}
  Errata ID       & JIRA-3079         \\
  Functional Area & System OS         \\
  Component       & NOC               \\
  Severity        & S5: No workaround
\end{tblr}

% after
\subsection{Software blows up after starting} % \label{JIRA-3079}

\begin{tblr}{colspec={>{\bfseries}ll}, column{1}={leftsep=0pt}}
  Errata ID       & JIRA-3079         \\
  Functional Area & System OS         \\
  Component       & NOC               \\
  Severity        & S5: No workaround
\end{tblr}
\end{document}
```

![image](https://github.com/lvjr/tabularray/assets/6376638/e6c7f619-ed3b-41cc-8313-b39dcec00a3c)

BTW for tables as simple as this one, the traditional `tabular` environment + `array` package is already capable of.

**Reply by russurquhart1** [reply]
Thanks for your help on this!! That did the trick. I'm in the middle of trying to fix formatting issues, based on what was coded there before I got here! I DO, however, want to be able to go through and simplify the LaTex coding  as much as possible! Thanks again for your help!
